subsys/bluetooth/services/Kconfig.dis

```diff
@@ -7,9 +7,24 @@
 
 menuconfig BT_GATT_DIS
 	bool "Enable GATT Device Information service"
+	default n
```

Comment:
`n` is the implicit default, so an explicit `default n` should never be used (sorry for not noticing this earlier and causing these extra review rounds)

Reply:
Done